app/config/sources.inc.php

```diff
@@ -437,8 +439,18 @@
             'it', 'kab', 'ko', 'pl', 'pt-BR', 'sq', 'sv-SE', 'uk', 'zh-TW',
         ],
     ],
+    'mozorg/internet-health/decentralization.lang' => [
+        'deadline' => '2017-05-24',
+        'flags'    => [
+            'opt-in' => ['all'],
+        ],
+        'priority'          => 3,
+        'supported_locales' => [
+          'de', 'es-ES', 'fr',
```

Comment:
Last time you added cy and kab from the start. Do you want to do it this time too?